package.json

```diff
@@ -89,6 +89,7 @@
     "fast-xml-parser": "~4.2.4",
     "formidable": "~2.1.2",
     "fullnode": "file:lib/fullnode",
+    "http-shutdown":"~1.2.2",
```

Comment:
I do not understand this move. Can you provide more insight on what is wrong on our current approach and why is this needed?